src/rmview/rmview.py

```diff
@@ -302,7 +302,7 @@ def connected(self, ssh):
     self.ssh = ssh
     self.viewer.setWindowTitle("rMview - " + ssh.hostname)
 
-    log.info("Detected %s", ssh.fullDeviceVersion)
+    log.info("Detected device: %s", ssh.fullDeviceVersion)
```

Comment:
Added to clarify this as the device version, since I also added an info-print for the detected SW version.